### ISSUE TITLE
🐛☠️ Dead server link leads to white screen of death

### ISSUE THREAD

**Issue by shnizzedy** [ChildMindInstitute/MindLogger-bug-reports#73]
**Describe the bug**
I was using a temporary endpoint (via [ngrok](http://ngrok.io/)) last night to test backend changes on my localhost. My computer rebooted for updates overnight, killing my temporary url. Since I was logged in through that endpoint on my phone, opening MindLogger now gives me an inescapable white screen.

**To Reproduce**
1. Set up a temporary instance of [Girder for Mindlogger](https://github.com/ChildMindInstitute/mindlogger-app-backend).
2. Log into that instance in a [MindLogger mobile app](https://mindlogger.org).
3. Close your mobile app.
4. Kill or disable your temporary backend endpoint.
5. Open your mobile app.
6. See a white expanse.

**Expected behavior**
I think the app should probably return to the login screen after a failed attempt to connect.

**Screengrab**
![white screen of death](https://user-images.githubusercontent.com/5974438/71188595-40043600-224f-11ea-815d-4d8d86f2953e.gif)

**Smartphone (please complete the following information):**
 - Device: iPhone XR
 - OS: iOS 13.3
 - MindLogger
 - Version 0.9.8
